### ISSUE TITLE
coupe -A+f may not write -R to gmt.history correctly

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1249]
**Full script that generated the error**

```
#!/usr/bin/env bash

gmt begin beachball-cross-section
gmt makecpt -Cjet -T0/30/10
gmt coupe -JX15c/-10c -Bxaf+l"Distance (km)" -Byaf+l"Depth (km)" -BWSen \
	-Sd2c -Aa113/33/117/33/90/100/0/30+f -Q << EOF
# lon lat depth mrr mtt mpp mrt mrp mtp iexp name
114 32.9 24  0.06 -2.48  2.42 -0.34  0.52 -5.10 25 X Y strike-slip fault
115 33.0  5 -1.55  0.13  1.42  0.24 -0.29 -0.38 24 X Y normal fault
116 33.1 13  2.41 -0.15 -2.26  1.03 -1.81  0.48 22 X Y reverse fault
EOF
gmt colorbar -DJMR+ml+o1.5c/0c -Bx+l"Depth (km)" -Np
gmt end show
```

**Full error message**

```
psscale [ERROR]: Syntax error: -DJ requires the -R option
```

In the coupe command, the plot region is automatically determined to be `0/cross-section_length/depmin/depmax`. That's why I don't need to add a `-R` option. But, gmt colorbar complains no `-R` found.

**Comment by seisman** [comment]
If I add another plotting command like `echo 300 300 | gmt plot -Sc2c -Gred` before `colorbar`, then it works.

**Comment by PaulWessel** [comment]
Will need to look in pscoupe and probably need to add a special R history entry from the automatic region so it is written to the gmt.history.

**Comment by seisman** [comment]
Closed by #1251
